.github/workflows/test.yml

```diff
@@ -224,6 +224,40 @@ jobs:
           targetPlatform: ${{ matrix.platform }}
           allowDirtyBuild: true
 
+      ###########################
+      #       Build (GZip)      #
+      ###########################
+      - name: "⚙️ Setup (GZip)"
+        if: env.MODULE == 'webgl'
+        run: |
+          sed -i -e 's/webGLCompressionFormat: .*$/webGLCompressionFormat: 1/' reference-project-test/ProjectSettings/ProjectSettings.asset
```

Comment:
Could you add a comment in here, saying what this does? For example: "Set WebGL compression format to gzip".